release_notes.py

```diff
@@ -144,23 +146,25 @@ def load_data(self, data):
     # create a release notes section for store (add or modified) - return None if found missing release notes
     def release_notes_section(self, store, title_prefix, current_server_version):
         res = ""
+        beta_rn_paths = list()
         if store:
             new_str = ""
             new_count = 0
             for path in store:
                 with open(path, 'r') as file_obj:
-                    print(' - adding release notes ({}) for file - [{}]... '.format(path, title_prefix), end='')
```

Comment:
why did you remove this?

Reply:
because debugging and understanding the log became not possible